source/sass/main.scss

```diff
@@ -325,13 +327,17 @@ button {
     .container {
       margin-top: 0;
       margin-bottom: 0;
-      display: flex;
-      height: 220px;
-      justify-content: space-around;
-      align-items: center;
+      min-height: 220px;
+
+      @media screen and (min-width: $bp-md) {
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+      }
     }
 
     .info-box {
```

Comment:
I'd suggest to apply top and bottom padding to `.more-info .container` and `margin-bottom` to `.info-box` only if it's not the last child (e.g., `.info-box:not(:last-child)`) . If we just do `margin: 1.5rem 0;` on all `.info-box` the space in between `.info-box` is doubled than their outer margin.